languages/ruby/lib/oso/polar/polar.rb

```diff
@@ -43,11 +46,31 @@ def clear
       # @param name [String]
       # @raise [PolarFileExtensionError] if provided filename has invalid extension.
       # @raise [PolarFileNotFoundError] if provided filename does not exist.
-      def load_file(name)
+      def load_file(name) # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
         raise PolarFileExtensionError unless ['.pol', '.polar'].include? File.extname(name)
```

Comment:
You removed `.pol` on the Python version but not here. I haven't checked Java yet, but we should ensure it's the same across all three.